src/e2e/puppeteer/helpers/snapshotSimulation.ts

```diff
@@ -0,0 +1,16 @@
+import { Page } from 'puppeteer'
+import { WindowEm } from '../../../initialize'
+
+const em = window.em as WindowEm
+
+/** Sets testFlags for simulating drag and drop process . */
+const simaulateSnapshot = async (page: Page): Promise<void> => {
```

Comment:
Let's call this `simulateDragAndDrop` and have it accept parameters `{ drag?: boolean, drop?: boolean }` so that we can set them independently.

We could also call it `setDragAndDropColorBlocking`. What do you think?

Please also rename the file to match whatever we decide.

Reply:
I think `simulateDragAndDrop` is a fine name for this function.